### PR TITLE
[MIG] web_view_google_map: Migration to 13.0

### PR DESCRIPTION
**Pull request by brian10048**
depends #287

### CODE REVIEW

File: web_view_google_map/views/res_partner.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_res_partner_google_map" model="ir.ui.view">
+        <field name="name">view.res.partner.google_map</field>
+        <field name="model">res.partner</field>
+        <field name="arch" type="xml">
+            <google_map class="o_res_partner_map" library="geometry" string="Google Map" lat="partner_latitude" lng="partner_longitude" colors="blue:company_type=='person';green:company_type=='company';">
```
Comment (AmmarOfficewalaSerpentCS) [inline web_view_google_map/views/res_partner.xml:7]: @brian10048 , 
I don't see any js widget for a google_map tag

**Review by AmmarOfficewalaSerpentCS — CHANGES_REQUESTED** [review]
Please change the code in JS for a google map tag.

File: web_view_google_map/views/res_partner.xml (hunk 1 above)
Comment (brian10048) [reply web_view_google_map/views/res_partner.xml:7]: Thank you for noticing @AmmarOfficewalaSerpentCS 
I updated the JS files for the change from `map` to `google_map`

**Comment by brian10048** [comment]
@yvaucher would you be able to look at this error with Travis? [https://travis-ci.com/github/OCA/geospatial/jobs/496758012#L580](https://travis-ci.com/github/OCA/geospatial/jobs/496758012#L580)

**Comment by yvaucher** [comment]
@brian10048 can you rebase?

I attempted to fix the CI there: https://github.com/OCA/geospatial/commit/6adf14cfb68a991312ff657c360865ba5e6ffb4a

**Comment by brian10048** [comment]
Thanks @yvaucher - It appears to be working on #287 and I will rebase this one next

**Pull request closed without merging**